packages/taco/src/conditions/schemas/context.ts

```diff
@@ -11,12 +11,17 @@ export const contextParamSchema = z
     `A Context Parameter i.e. a placeholder used within conditions and specified at the encryption time, whose value is provided at decryption time.`,
   );
 
-const paramSchema = z.union([plainStringSchema, z.boolean(), z.number()]);
+const paramSchema = z.union([
+  plainStringSchema,
+  z.boolean(),
+  z.number(),
+  z.bigint(),
+]);
 
 const nonFloatParamSchema = z
-  .union([plainStringSchema, z.boolean(), z.number().int()])
+  .union([plainStringSchema, z.boolean(), z.number().int(), z.bigint()])
```

Comment:
I think we need to refine this bigint, because it shouldn't allow values > uint256 max or values < int256 min for onchain conditions - see https://github.com/derekpierre/taco-web/blob/large-number-tests/packages/taco/src/conditions/schemas/context.ts#L35. 

That may beg the follow-up question as to whether `nonFloatParamSchema` is still the best name for this schema.

Reply:
Good point!
Also checked for the int number to be in the same range.